api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/authentication/LoginController.java

```diff
@@ -0,0 +1,73 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.authentication;
+
+import edu.cornell.mannlib.vitro.webapp.beans.UserAccount;
+import edu.cornell.mannlib.vitro.webapp.controller.VitroHttpServlet;
+import edu.cornell.mannlib.vitro.webapp.controller.authenticate.BasicAuthenticator;
+import edu.cornell.mannlib.vitro.webapp.dynapi.RESTEndpoint;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.jena.base.Sys;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet(name = "Login", urlPatterns = {"/rest_login"})
```

Comment:
Can we follow the existing dynamic API request paths?

https://github.com/vivo-project/Vitro/blob/sprint-dynapi-2022-feb-staging/api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/RESTEndpoint.java#L27

I would suggest `/api/login` and possibly create constant such as https://github.com/vivo-project/Vitro/blob/sprint-dynapi-2022-feb-staging/api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/request/ApiRequestPath.java#L15

```
API_SERVLET_PATH + "/login";
```